### DISCUSSION TITLE
How to troubleshoot when an email is not delivered?

### DISCUSSION THREAD

**Discussion by yamrzou** [emersion/maddy#572] [Q&A]
Hi,

The mail server is working great, except a specific Gmail address not receiving an email sent via Maddy even though our logs show "queue: delivered". Other Gmail addresses work fine.

How to find out what's going on?

**Comment by foxcpp** [comment]
"Default" list of things to check when you have delivery problems:
1. Is DKIM check passing? 
2. Is SPF check passing?
3. Is DMARC check passing?

All that info should be visible in Authorization-Results header in any message you received on gmail from maddy (you should see "pass" for all of them).
Next up, make sure you have rDNS record setup for the IP you use for sending.

If everything is fine, then double-check that the message is not in the Spam folder. It is hard to diagnose issues with gmail since their policy is opaque and they are known for silently dropping messages that fail some checks - which is what might be happening here.
